### PR TITLE
2 more Strange objects

### PR DESCRIPTION
**Pull request by Time-Green**
I got kinda bored of this project, so I just added 2.

Cutter:
This object starts to fucking cut open your belly and rip out your internal organs. You got about 4 seconds to drop it. 

If it finishes cutting your belly open, it will cut your head off.
Fun

HumanClone:
Basicly, turns the object into an exact clone of you. It's still just an object, so you can carry yourself around.

strangeobjectslauggter
(The living Tomato is fake)

### CODE REVIEW

File: code/modules/research/experimentor.dm [hunk 1]
```diff
@@ -707,3 +708,57 @@
...
+			H << "<span class='warning'>[src] begins violently tearing at your [BP.name]!</span>"
+			while((BP.brute_dam) < 40) //Approximately 4 seconds. Can be pretty lethal.
+				if(src.loc == user)
+					BP.brute_dam += 4
+					sleep(4)
+					playsound(src,'sound/weapons/bladeslice.ogg',40,1) //This'll better fucking alert you.
+				else
+					return //This will lag extremely bad if you don't.
+			note = 0
+			if(rand(60))
```
Comment (ShadowDeath6) [inline code/modules/research/experimentor.dm]: if this is what I think it is then use prob

File: code/modules/research/experimentor.dm [hunk 2]
```diff
@@ -707,3 +708,57 @@
...
+/obj/item/weapon/relic/proc/cutter(mob/user)
+	var/mob/living/carbon/human/H = user
+	if(ishuman(H))
+		for(var/obj/item/bodypart/BP in H.bodyparts)
+			H << "<span class='warning'>[src] begins violently tearing at your [BP.name]!</span>"
+			while((BP.brute_dam) < 40) //Approximately 4 seconds. Can be pretty lethal.
+				if(src.loc == user)
+					BP.brute_dam += 4
+					sleep(4)
+					playsound(src,'sound/weapons/bladeslice.ogg',40,1) //This'll better fucking alert you.
```
Comment (ShadowDeath6) [inline code/modules/research/experimentor.dm:741]: give text message as well

File: code/modules/research/experimentor.dm [hunk 3]
```diff
@@ -707,3 +708,57 @@
...
+		overlays.Cut()
+		name = realName
+		throwSmoke(get_turf(src))
+		note = 0
+
+
+/obj/item/weapon/relic/proc/cutter(mob/user)
+	var/mob/living/carbon/human/H = user
+	if(ishuman(H))
+		for(var/obj/item/bodypart/BP in H.bodyparts)
```
Comment (ShadowDeath6) [inline code/modules/research/experimentor.dm:734]: this will kill you immensely fast as it does damage to every body part every time it runs

Comment (Time-Green) [reply code/modules/research/experimentor.dm:734]: It doesn't

File: code/modules/research/experimentor.dm [hunk 4]
```diff
@@ -707,3 +708,57 @@
...
+			while((BP.brute_dam) < 40) //Approximately 4 seconds. Can be pretty lethal.
+				if(src.loc == user)
+					BP.brute_dam += 4
+					sleep(4)
+					playsound(src,'sound/weapons/bladeslice.ogg',40,1) //This'll better fucking alert you.
+				else
+					return //This will lag extremely bad if you don't.
+			note = 0
+			if(rand(60))
+				BP.dismember(BRUTE)
```
Comment (ShadowDeath6) [inline code/modules/research/experimentor.dm:746]: dismembering chest or head is an instant kill just so you're aware

Comment (Time-Green) [reply code/modules/research/experimentor.dm:746]: I know. And exploding from the other strange object also kills you.
But this gives you an actual obvious warning.

**Comment by ShadowDeath6** [comment]
tl;dr I don't like artifacts that kill you

clone is ok but kinda odd since it's an item but _shrug_

File: code/modules/research/experimentor.dm (hunk 1 above)
Comment (Time-Green) [reply code/modules/research/experimentor.dm]: Woops

**Comment by Super3222** [comment]
Cutter is stupid and unfun. Clone is ~~from regular show~~ an okay idea.

File: code/modules/research/experimentor.dm (hunk 3 above)
Comment (Super3222) [reply code/modules/research/experimentor.dm:734]: for(var/obj/item/bodypart/BP in H.bodyparts)

File: code/modules/research/experimentor.dm [hunk 5]
```diff
@@ -707,3 +708,57 @@
...
+			if(prob(60))
+				BP.dismember(BRUTE)
+				playsound(get_turf(src), 'sound/misc/splort.ogg', 50, 1, -1)
+				warn_admins(user, "Cutter")
+			else
+				H << "<span class='warning'>[src] shuts down, you got lucky!</span>"
+				return 0
+	else
+		H << "<span class='notice'>[src] judges you unworthy.</span>" //Should appear to everything that doesn't support dismembering.
+																	     //NODISMEMBER, monkeys, drones, etc
```
Comment (Super3222) [inline code/modules/research/experimentor.dm:754]: thats a lot of space

**Comment by ghost** [comment]
wow this is dumb

maybe we should just remove the expermemeor

**Comment by Time-Green** [comment]
If you guys want I can have it pick a random limb to dismember.

File: code/modules/research/experimentor.dm (hunk 5 above)
Comment (Super3222) [reply code/modules/research/experimentor.dm:754]: No really. You need to backspace all of that

**Comment by Super3222** [comment]
Sure. And making it "heat up" before it cuts off your arm doesn't make sense. It should be like "a motor within the device starts churning..."

File: code/modules/research/experimentor.dm (hunk 5 above)
Comment (Time-Green) [reply code/modules/research/experimentor.dm:754]: Oh, like that. Sure.

**Comment by Time-Green** [comment]
It doesn't heat up, it gives a message:
THE OBJECT STARTS STABBING YOUR CHEST or something.

Edit: I'll fix stuff tomorrow.
